### ISSUE TITLE
Cannot debug adios under Matlab anymore because of TAU asserts

### ISSUE THREAD

**Issue by pnorbert** [ornladios/ADIOS2#1318]
I am trying to debug the Matlab bindings, which is an involved [manual process](https://www.mathworks.com/help/matlab/matlab_external/debugging-on-linux-platforms.html) under matlab and gdb. However, now calling an adios function will cause the debugger to abort:

```
>> dbmex on                        
>> f=adiosopen('heat.bp', 'Verbose', 4)

CallArguments =

adiosopenc:
  File name=/autofs/nccs-svm1_proj/e2e/pnorbert/ADIOS/ADIOS2/bindings/Matlab/test/heat.bp 
  Verbose=4 

Assertion 'mytid == mypid' failed, file '/ccs/proj/e2e/pnorbert/ADIOS/ADIOS2/source/adios2/toolkit/profiling/taustubs/tautimer.cpp' line '213'.
Program received signal SIGABRT, Aborted.
```

**Comment by khuck** [comment]
Interesting...easy enough to fix, I'll do that

**Comment by eisenhauer** [comment]
You debug and I’ll merge the PR...

**Comment by khuck** [comment]
@pnorbert @eisenhauer see #1319

**Comment by eisenhauer** [comment]
Merged #1319 .  @pnorbert, please confirm that this fixes the problem.

**Comment by eisenhauer** [comment]
The fix was merged, after many months I'll assume that the problem is gone.
